Index: test/com/thoughtworks/proxy/AllTests.java
===================================================================
--- test/com/thoughtworks/proxy/AllTests.java	(revision 69)
+++ test/com/thoughtworks/proxy/AllTests.java	(working copy)
@@ -12,6 +12,7 @@
 
 import com.thoughtworks.proxy.factory.CglibProxyFactory;
 import com.thoughtworks.proxy.factory.StandardProxyFactory;
+import com.thoughtworks.proxy.toys.autopool.AutoPoolTest;
 import com.thoughtworks.proxy.toys.decorate.DecoratingTest;
 import com.thoughtworks.proxy.toys.delegate.DelegatingTest;
 import com.thoughtworks.proxy.toys.echo.EchoingTest;
@@ -63,6 +64,7 @@
 		TestSuite suite = new TestSuite("Tests using " + type + "ProxyFactory");
 		ProxyTestCase.PROXY_FACTORY = factory;
         suite.addTestSuite(ProxyFactoryTest.class);
+        suite.addTestSuite(AutoPoolTest.class);
         suite.addTestSuite(DecoratingTest.class);
 		suite.addTestSuite(DelegatingTest.class);
 		suite.addTestSuite(EchoingTest.class);
